homeassistant/components/brottsplatskartan/sensor.py

```diff
@@ -29,9 +30,11 @@ async def async_setup_entry(
     app = entry.data[CONF_APP_ID]
     name = entry.title
 
-    bpk = BrottsplatsKartan(app=app, area=area, latitude=latitude, longitude=longitude)
+    bpk = BrottsplatsKartan(
+        app=app, areas=[area], latitude=latitude, longitude=longitude
```

Comment:
It's not valid to pass `None` in the list of areas to the library.

https://github.com/chrillux/brottsplatskartan/blob/5975e4bf2a9cb4a5e19518f57a6934e90b6dbb0b/brottsplatskartan/__init__.py#L34-L36

https://github.com/chrillux/brottsplatskartan/blob/5975e4bf2a9cb4a5e19518f57a6934e90b6dbb0b/brottsplatskartan/__init__.py#L11-L18

Reply:
```suggestion
        app=app, areas=[area] if areas else None, latitude=latitude, longitude=longitude
```

Reply:
Good catch!